src/docs/sdk/performance/index.mdx

```diff
@@ -219,11 +219,21 @@ Which in reality is useful for proxies to set it to `0` and opt out of tracing.
 ## Static API Changes
 
 The `Sentry.startTransaction` function should take two arguments - the `transactionContext` passed to the `Transaction` constructor and an optional `customSamplingContext` object containing data to be passed to `tracesSampler` (if defined).
-
 It creates a `Transaction` bound to the current hub and returns the instance.
 Users interact with the instance for creating child spans and, thus, have to
 keep track of it themselves.
 
+With `Sentry.span` users can attach spans to an already ongoing transaction.
+This property returns a `SpanProtocol` if a running transaction is bound to
```

Comment:
I believe `SpanProtocol` is specific to Cocoa.

Also, there's a section just below here in the same file titled `Scope Changes` that describes `setSpan`.  I think this section belongs there, and can use similar generic terminology as that paragraph.